log/common.go

```diff
@@ -0,0 +1,36 @@
+package log
+
+import (
+	"io"
+	"os"
+)
+
+// For debugging purposes we can change the output-writer
+var stdOut io.Writer
+var stdErr io.Writer
+
+func init() {
+	stdOut = os.Stdout
+	stdErr = os.Stderr
+}
+
+const (
+	lvlWarning = iota - 20
+	lvlError
+	lvlFatal
+	lvlPanic
+	lvlInfo
+	lvlPrint
+)
+
+// Format indicates how the prints will be shown
+var Format = FormatLvl
```

Comment:
it seems that whatever the `Format` is, `Lvl{1,2,3,4,5}` will always use the `FormatLvl` format,meaning with the name of the function, line etc.
So if people wants to use `Info,Warn...` they have to explicitly change the format. Am I missing something ?
If not, it may be best to directly put the format to `FormatPython` or `FormatNone`, so people don't have to change it.